pkgs/servers/web-apps/livebook/default.nix

```
@@ -0,0 +1,45 @@
+# Created with inspiration from https://github.com/hauleth/nix-elixir/blob/master/pkgs/livebook.nix
+
+{ lib, beamPackages, makeWrapper, rebar3, elixir, erlang, fetchFromGitHub }:
+beamPackages.mixRelease rec {
+  pname = "livebook";
+  version = "0.9.2";
+
+  inherit elixir;
+
+  buildInputs = [ erlang ];
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  src = fetchFromGitHub {
+    owner = "livebook-dev";
+    repo = "livebook";
+    rev = "v${version}";
+    hash = "sha256-khC3gtRvywgAY6qHslZgAV3kmziJgKhdCB8CDg/HkIU=";
+  };
+
+  mixFodDeps = beamPackages.fetchMixDeps {
+    pname = "mix-deps-${pname}";
+    inherit src version;
+    sha256 = "rwWGs4fGeuyV6BBFgCyyDwKf/YLgs1wY0xnHYy8iioE=";
+  };
+
+  installPhase = ''
+    mix escript.build
+    mkdir -p $out/bin
+    mv ./livebook $out/bin
+
+    wrapProgram $out/bin/livebook \
+      --suffix PATH : ${lib.makeBinPath [ elixir ]} \
```

```suggestion
      --prefix PATH : ${lib.makeBinPath [ elixir ]} \
```
shouldn't we rather prefix here? Using another elixier from PATH could presumable break the application?

---

I'm not sure what the difference is, but I assume you're right :-)